kyaml/fn/runtime/runtimeutil/runtimeutil_test.go

```diff
@@ -1379,3 +1379,52 @@ metadata:
 		assert.Equal(t, tc.required, fn.Container.Network.Required)
 	}
 }
+
+func Test_StringToStorageMount(t *testing.T) {
```

Comment:
Can we test the behavior of "kustomize fn run ./d --mount type=bind,src,dst=/tmp/source/" here?

Reply:
@prachirp,
I'm not sure if it can be included now, because kustomize's go.mod has to be updated first. 
I think it will fail if we add it right now.
Kustomize is using the specific version of kyaml. probably it's up to release-management of kustomize project.
It can be tested now locally though, if you modify the go.mod by yourself:
```
$ git clone https://github.com/kubernetes-sigs/kustomize.git
$ cd kustomize/kustomize
$ go get sigs.k8s.io/kustomize/kyaml@master
go: sigs.k8s.io/kustomize/kyaml master => v0.4.1-0.20200713212920-21a0fd33a285
$ go build main.go -o kstmz
$ cat fn/example-use.yaml # need to create this catalog and file
apiVersion: v1
kind: ConfigMap
metadata:
  name: my-config
  annotations:
    config.k8s.io/function: |
      container:
        image:  gcr.io/kpt-functions/kustomize-build
    config.kubernetes.io/local-config: 'true'
data:
  path: /tmp/source/
  '--reorder': none
$ cat test/kustomization.yaml # need to create this catalog and file
resources:
- x.yaml
$ cat test/x.yaml # need to create this file
apiVersion: v1
data:
  x: true
kind: ConfigMap
metadata:
  name: my-xxx
$ ./kstmz fn run ./fn --enable-exec --mount type=bind,src=$(pwd)/test/,dst=/tmp/source/
$ cat fn/configmap_my-xxx.yaml # this file has been generated!
apiVersion: v1
data:
  x: true
kind: ConfigMap
metadata:
  name: my-xxx
```
In parallel console I ran this
```
$ ps aux| grep "docker "
ubuntu   27156  3.0  0.1 1154712 63452 pts/0   Sl+  23:44   0:00 docker run --rm
-i -a STDIN -a STDOUT -a STDERR --network none --user nobody --security-opt=no-
new-privileges --mount type=bind,source=/home/ubuntu/kpt-functions-catalog/funct
ions/ts/test/,target=/tmp/source/,readonly -e LS_COLORS (...some -e skipped...) -e _ -e LOG_TO_STDERR -e
STRUCTURED_RESULTS 6d035c6e0c9a
ubuntu   27178  0.0  0.0  14856  1068 pts/2    S+   23:44   0:00 grep --color=au
to docker
```
as you can see mount now has right args and that's why you see that configmap_my-xxx.yaml has been generated!

Reply:
Thanks, I see the grep "docker" command with the same args locally.